src/main/java/com/ericsson/ei/frontend/utils/EIRequestsControllerUtils.java

```diff
@@ -90,24 +90,40 @@ private String getBackEndUrl(String requestQuery) throws EiBackendInstancesExcep
             List<NameValuePair> params = getParameters(requestQuery);
             if (requestQuery.contains(BACKEND_URL_KEY_NAME)) {
                 requestUrl = extractUrlFromParameters(params);
-            } else {
-                String backEndName = extractBackEndNameFromParameters(params);
-                BackEndInformation backEndInformation = backEndInstancesUtils.getBackEndInformationByName(backEndName);
-                if (backEndInformation == null) {
-                    throw new EiBackendInstancesException(
-                            "EI Backend instance '" + backEndName +  "' does not exist in EI Backend Instances list.");
+            } else if(requestQuery.contains(BACKEND_NAME_KEY_NAME)) {
+                try {
+                    requestUrl = getUrlFromBackendInstancesUtils(params);
```

Comment:
Imagine the scenaria when a user selects a back-end that does not exist and he creates a subscription in the default one and not the selected one. Should this be possible?

Reply:
That is how the functionality worked before. I didn't want to change too much in the behaviour in the refactoring.